app/assets/javascripts/manage_submissions.js

```diff
@@ -1,74 +1,204 @@
-$(document).ready(function() {
+const manage_submissions_endpoints = {
+  'regrade-selected': 'regradeBatch',
+};
 
-  // USE LATER FOR GROUPING ROWS (POSSIBLY):
+$(document).ready(function() {
 
-  // $.fn.dataTable.ext.search.push(
-  //   function(settings, data, dataIndex) {
-  //     var filterOnlyLatest = $("#only-latest").is(':checked');
-  //     if (!filterOnlyLatest) {
-  //       // if not filtered, return all the rows
-  //       return true;
-  //     } else {
-  //       var isSubmissionLatest = data[8]; // use data for the age column
-  //       return (isSubmissionLatest == "true");
-  //     }
-  //   }
-  // );
+  var selectedStudentCids = [];
+  var selectedSubmissions = [];
 
   var table = $('#submissions').DataTable({
-    "dom": 'fBrt', // show buttons, search, table
-    buttons: [
-      { text: '<i class="material-icons">cached</i>Regrade Selected', className: 'btn submissions-selected disabled' },
-      { text: '<i class="material-icons">delete_outline</i>Delete Selected', className: 'btn submissions-selected disabled' },
-      { text: '<i class="material-icons">download</i>Download Selected', className: 'btn submissions-selected disabled' },
-      { text: '<i class="material-icons">done</i>Excuse Selected', className: 'btn submissions-selected disabled' }
-    ]
+    'dom': 'f<"selected-buttons">rt', // show buttons, search, table
+    'paging': false,
   });
 
-  var selectedSubmissions = [];
+  // SELECTED BUTTONS
 
-  // USE LATER FOR REGRADE SELECTED (POSSIBLY):
+  // create selected buttons inside datatable wrapper
+  var regradeHTML = document.getElementById('regrade-batch-html').innerHTML;
```

Comment:
nit: for consistency, use jquery (e.g., `$('#regrade-batch-html').html();`)

Reply:
Fixed.